### ISSUE TITLE
Using the Qwen2.5-Math-1.5B model with the Math500 dataset and the latest version of Verl code, the validation result is 0.608. Shouldn't it be around 0.3?

### ISSUE THREAD

**Issue by Tgzz666** [verl-project/verl#1268]
sh:
 PYTHONUNBUFFERED=1 python3 -m verl.trainer.main_ppo \
 data.train_files=/data1//dataset/train/deepscaler_uniform_train.parquet \
 data.val_files=/data1//dataset/test/math_test.parquet \
 data.train_batch_size=256 \
 data.max_prompt_length=1024 \
 data.filter_overlong_prompts=False \
 data.max_response_length=2048 \
 data.shuffle=True \
 actor_rollout_ref.model.path=/data1//models/Qwen2.5-Math-1.5B \
 actor_rollout_ref.actor.optim.lr=1e-6 \
 actor_rollout_ref.actor.ppo_mini_batch_size=64 \
 actor_rollout_ref.actor.ppo_micro_batch_size_per_gpu=1 \
 actor_rollout_ref.rollout.log_prob_micro_batch_size_per_gpu=2 \
 actor_rollout_ref.rollout.tensor_model_parallel_size=2 \
 actor_rollout_ref.rollout.gpu_memory_utilization=0.4 \
 actor_rollout_ref.ref.log_prob_micro_batch_size_per_gpu=4 \
 critic.optim.lr=1e-5 \
 critic.model.path=/data1//models/Qwen2.5-Math-1.5B \
 critic.ppo_micro_batch_size_per_gpu=4 \
 algorithm.kl_ctrl.kl_coef=0.001 \
 trainer.logger=['console','wandb'] \
 trainer.project_name='verl_deepscaler' \
 trainer.experiment_name='math1.5B-base-8epoch' \
 trainer.val_before_train=True \
 trainer.default_hdfs_dir=null \
 trainer.n_gpus_per_node=4 \
 trainer.nnodes=1 \
 trainer.save_freq=-1 \
 trainer.test_freq=2 \
 trainer.log_val_generations=30 \
 trainer.total_epochs=8 2>&1 | tee verl_demo.log$@

val result:
(WorkerDict pid=457904) WARNING 04-26 16:27:03 [topk_topp_sampler.py:63] FlashInfer is not available. Falling back to the PyTorch-native implementation of top-p & top-k sampling. For the best performance, please install FlashInfer. [repeated 3x across cluster]
(WorkerDict pid=458234) kwargs: {'n': 1, 'logprobs': 0, 'max_tokens': 2048, 'detokenize': False, 'temperature': 1.0, 'top_k': -1, 'top_p': 1, 'ignore_eos': False} [repeated 3x across cluster]
(TaskRunner pid=457401) test_gen_batch meta info: {'eos_token_id': 151643, 'pad_token_id': 151643, 'recompute_log_prob': False, 'do_sample': False, 'validate': True}
(TaskRunner pid=457401) validation generation end
(TaskRunner pid=457401) [prompt] system
(TaskRunner pid=457401) Please reason step by step, and put your final answer within \boxed{}.
(TaskRunner pid=457401) user
(TaskRunner pid=457401) Convert the point $(0,3)$ in rectangular coordinates to polar coordinates.  Enter your answer in the form $(r,\theta),$ where $r > 0$ and $0 \le \theta < 2 \pi.$ Let's think step by step and output the final answer within \boxed{}.
(TaskRunner pid=457401) assistant
(TaskRunner pid=457401) 
(TaskRunner pid=457401) [response] To convert the point \((0, 3)\) from rectangular coordinates to polar coordinates, we need to find the radius \(r\) and the angle \(\theta\).
(TaskRunner pid=457401) 
(TaskRunner pid=457401) 1. **Radius \(r\)**:
(TaskRunner pid=457401)    The radius \(r\) is the distance from the origin to the point \((0, 3)\). It can be calculated using the Pythagorean theorem:
(TaskRunner pid=457401)    \[
(TaskRunner pid=457401)    r = \sqrt{x^2 + y^2}
(TaskRunner pid=457401)    \]
(TaskRunner pid=457401)    where \(x = 0\) and \(y = 3\).
(TaskRunner pid=457401) 
(TaskRunner pid=457401) 2. **Angle \(\theta\)**:
(TaskRunner pid=457401)    The angle \(\theta\) is the angle formed with the positive x-axis. It can be calculated using the arctangent function:
(TaskRunner pid=457401)    \[
(TaskRunner pid=457401)    \theta = \arctan\left(\frac{y}{x}\right)
(TaskRunner pid=457401)    \]
(TaskRunner pid=457401)    However, since \(x = 0\), we need to consider the special case where the point lies on the y-axis. In this case, \(\theta = \frac{\pi}{2}\) if \(y > 0\) and \(\theta = \frac{3\pi}{2}\) if \(y < 0\).
(TaskRunner pid=457401) 
(TaskRunner pid=457401) Let's calculate these values using Python.
(TaskRunner pid=457401) ```python
(TaskRunner pid=457401) import math
(TaskRunner pid=457401) 
(TaskRunner pid=457401) # Given rectangular coordinates
(TaskRunner pid=457401) x = 0
(TaskRunner pid=457401) y = 3
(TaskRunner pid=457401) 
(TaskRunner pid=457401) # Calculate radius r
(TaskRunner pid=457401) r = math.sqrt(x**2 + y**2)
(TaskRunner pid=457401) 
(TaskRunner pid=457401) # Calculate angle theta
(TaskRunner pid=457401) # Since x = 0, we need to handle the special case
(TaskRunner pid=457401) if x == 0:
(TaskRunner pid=457401)     if y > 0:
(TaskRunner pid=457401)         theta = math.pi / 2
(TaskRunner pid=457401)     else:
(TaskRunner pid=457401)         theta = 3 * math.pi / 2
(TaskRunner pid=457401) else:
(TaskRunner pid=457401)     theta = math.atan2(y, x)
(TaskRunner pid=457401) 
(TaskRunner pid=457401) # Ensure theta is in the range [0, 2*pi)
(TaskRunner pid=457401) if theta < 0:
(TaskRunner pid=457401)     theta += 2 * math.pi
(TaskRunner pid=457401) 
(TaskRunner pid=457401) print(((r, theta)))
(TaskRunner pid=457401) ```
(TaskRunner pid=457401) ```output
(TaskRunner pid=457401) (3.0, 1.5707963267948966)
(TaskRunner pid=457401) ```
(TaskRunner pid=457401) The polar coordinates for the point \((0, 3)\) are \((3.0, 1.5707963267948966)\).
(TaskRunner pid=457401) 
(TaskRunner pid=457401) In terms of \(\pi\), this is \((3, \frac{\pi}{2})\).
(TaskRunner pid=457401) 
(TaskRunner pid=457401) Thus, the final answer is:
(TaskRunner pid=457401) \[
(TaskRunner pid=457401) \boxed{(3, \frac{\pi}{2})}
(TaskRunner pid=457401) \]
(TaskRunner pid=457401) [ground_truth] \left( 3, \frac{\pi}{2} \right)
(TaskRunner pid=457401) [score] 1.0
(TaskRunner pid=457401) ('Initial validation metrics: '
(TaskRunner pid=457401)  "{'val-core/DigitalLearningGmbH/MATH-lighteval/reward/mean@1': 0.608, "
(TaskRunner pid=457401)  "'val-aux/DigitalLearningGmbH/MATH-lighteval/reward/std@1': 0.0, "
(TaskRunner pid=457401)  "'val-core/DigitalLearningGmbH/MATH-lighteval/reward/best@1/mean': 0.608, "
(TaskRunner pid=457401)  "'val-core/DigitalLearningGmbH/MATH-lighteval/reward/best@1/std': 0.0, "
(TaskRunner pid=457401)  "'val-aux/DigitalLearningGmbH/MATH-lighteval/reward/worst@1/mean': 0.608, "
(TaskRunner pid=457401)  "'val-aux/DigitalLearningGmbH/MATH-lighteval/reward/worst@1/std': 0.0}")
(TaskRunner pid=457401) step:0 - val-core/DigitalLearningGmbH/MATH-lighteval/reward/mean@1:0.608 - val-aux/DigitalLearningGmbH/MATH-lighteval/reward/std@1:0.000 - val-core/DigitalLearningGmbH/MATH-lighteval/reward/best@1/mean:0.608 - val-core/DigitalLearningGmbH/MATH-lighteval/reward/best@1/std:0.000 - val-aux/DigitalLearningGmbH/MATH-lighteval/reward/worst@1/mean:0.608 - val-aux/DigitalLearningGmbH/MATH-lighteval/reward/worst@1/std:0.000

**Comment by Kirrito-k423** [comment]
### Scheduled issue cleanup / 定时 issue 清理

中文：该 issue 命中定时清理规则：创建超过 6 个月且创建后没有任何评论或公开引用。因此按定时清理标准先关闭。若作者或社区成员仍能提供新的复现、证据或需求背景，欢迎继续评论请求重开，或创建带有最新信息的新 issue。

证据：
- Issue: https://github.com/verl-project/verl/issues/1268
- 该 issue 创建于 `2025-04-26T08:58:28Z` UTC，comments 数量为 0；timeline 检查未发现公开引用、PR/issue cross-reference、commit/PR mention 或其它公开关联互动；截至 2026-07-05 创建时间已超过 6 个月。
- 已检查 comments 来源：https://api.github.com/repos/verl-project/verl/issues/1268/comments
- 已检查 timeline 来源：https://api.github.com/repos/verl-project/verl/issues/1268/timeline

English: This issue matches the scheduled cleanup rule: it is older than 6 months and has had no comments or public references since creation. Therefore, this is being closed under the scheduled cleanup policy. If the author or community members can still provide a fresh reproduction, evidence, or requirement context, please comment to request reopening or open a new issue with updated information.

Evidence:
- Issue: https://github.com/verl-project/verl/issues/1268
- This issue was created at `2025-04-26T08:58:28Z` UTC, has 0 comments, and the timeline check found no public references, PR/issue cross-references, commit/PR mentions, or other public linked interactions; it is more than 6 months old as of 2026-07-05.
- Checked comments source: https://api.github.com/repos/verl-project/verl/issues/1268/comments
- Checked timeline source: https://api.github.com/repos/verl-project/verl/issues/1268/timeline

This response was generated by a scheduled Codex GPT5.5 issue processing task.
